test/x/exercise_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require_relative '../x_helper'
+
+module X
+  class ExerciseTest < Minitest::Test
+    def test_test_files
+      f= './test/fixtures/xapi_v3_exercise_tests.json'
+      Xapi.stub(:get, [200, File.read(f)]) do
+        _, response = Exercise::TestFiles.find('ruby', 'saddle-points')
+        expected = response['exercise']
+
+        assert_equal 'Ruby', expected['language']
+        assert_equal 'Saddle Points', expected['name']
+        assert_kind_of Hash, expected['files']
```

Comment:
If we use the fake data by running x-api in test mode, then we can assert on some of the real values here.